src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  width: calendarWidth(100, 1, 10) + px;
```

Comment:
What are 100, 1 and 10? I think it's better to create variables for these values, or put them inside the function